Fortinet/fortiproxy/ingest/parser.yml

```diff
@@ -42,12 +42,11 @@ stages:
           event.start: '{{parsed_date.timestamp[:-6]}}Z'
           action.name: '{{parsed_event.message.action}}'
           action.type: '{{parsed_event.message.utmaction}}'
-          host.name: '{{parsed_event.message.to}}'
+          host.name: '{{parsed_event.message.to or parsed_event.message.srcname}}'
```

Comment:
Not sure this will work if `message.to` does not exist, have you tested it ?

Reply:
We have others parsers with the same pattern. It's one of our regular tricks for parsers.